src/test/java/uk/gov/hmcts/reform/divorce/orchestration/service/EmailServiceTest.java

```
@@ -67,13 +65,38 @@ public void sendEmailShouldNotPropagateNotificationClientException()
         throws NotificationClientException {
         doThrow(new NotificationClientException(new Exception("Exception inception")))
             .when(mockClient).sendEmail(anyString(), anyString(), eq(null), anyString());
-        try {
-            emailService.sendEmail(EMAIL_ADDRESS,
+
+        //no exception thrown
+        emailService.sendEmail(EMAIL_ADDRESS,
```

I've just realised we're swallowing the error in this method, tbh It should be in the other way. By default propagate the error and have a method that explicitly swallow the error

---

nvm, this probably needs to verify the error  from response

---

Yeah we should try to use that less..